cmd/tools/genrpcwrappers/main.go

```diff
@@ -291,7 +291,8 @@ func makeGetMatchingClient(reqType reflect.Type) string {
 	case "UpdateTaskQueueUserDataRequest",
 		"ReplicateTaskQueueUserDataRequest",
 		"RecordWorkerHeartbeatRequest",
-		"ListWorkersRequest":
+		"ListWorkersRequest",
+		"UpdateTaskQueueConfigRequest":
```

Comment:
This is the wrong place, this one should be routed to the root WF partition, in the group with UpdateWorkerVersioningRulesRequest